tests/testthat/test-finalizer.R

```diff
@@ -1,4 +1,4 @@
-context("finalizer")
```

Comment:
I think it makes sense to just remove all of these instead of commenting them out.

Reply:
Done!